src/web100srv.c

```diff
@@ -2657,14 +2123,10 @@ int main(int argc, char **argv) {
   // These flags keep track of running processes.  The 'testing' flag
   // indicates a test is currently being performed.  The 'waiting' counter
   // shows how many tests are pending.
-  testing = 0;
-  mclients = 0;
-  waiting = 0;
-  loopcnt = 0;
-  head_ptr = NULL;
-  sig17 = 0;
-  sem_init(&ndtq, 0, 1);
-  NDT_server_main_loop(ssl_context, listenfd);
+  pipe(tmp_pipe_fds);
+  signalfd_read = tmp_pipe_fds[0];
+  signalfd_write = tmp_pipe_fds[1];
```

Comment:
https://www.youtube.com/watch?v=WdNsltQXTVU#t=8

I will add the prefix `global_`, because global variables should not be subtle.